content/rc/security/single-sign-on/saml-sso.md

```diff
@@ -160,13 +151,13 @@ To bind other accounts to an existing SAML SSO configuration:
 
 ## Deactivate SAML SSO
 
-A SAML-enabled account must have a local (non-SAML) user with the owner role assigned before you can deactivate SAML SSO for that account.
+Before you can deactivate SAML SSO for an account, it must have a local (non-SAML) user with the owner role assigned. You can only deactivate SAML SSO for the account you are currently signed in to.
```

Comment:
@rrelledge to deactivate SAML you must be logged in to that specific account as a local (non-SAML) user with the owner role, i.e. even if the account has another local owner user but you are signed in as a SAML user you can still not deactivate it.

Reply:
Fixed.